### PR TITLE
Add support for weather stations

### PR DESCRIPTION
**Pull request by BlackSmith**
This PR adds support for these weather stations:
- Venus W174/W132
- Auriol H13726   (tested)
- Hama EWS 1500
- Meteoscan W155/W160

I tested this script on ESP-WROOM-32.

### CODE REVIEW

**Comment by 1technophile** [comment]
Thanks for this PR!

File: platformio.ini [hunk 1]
```diff
@@ -33,6 +33,7 @@ extra_configs =
 ;default_envs = esp32dev-all
 ;default_envs = esp32dev-rf
 ;default_envs = esp32dev-pilight
+;default_envs = esp32dev-weatherstation
```
Comment (1technophile) [inline platformio.ini]: If the code works with esp8266 and/or arduino would it be possible to add the corresponding environment definitions?

**Comment by BlackSmith** [comment]
I added environment definitions for eps82266 and arduino. 
I have tested esp8266.
I tried to test Arduino UNO with mini W5100, but arduino can not connect to MQTT server (error rc=5 on arduino and "Socket error on client <unknown>, disconnecting." on MQTT server). 
Maybe hardware issue on my W5100. :(

**Comment by 1technophile** [comment]
Thanks, It could be an issue also with the arduino having not enough memory to run the program correctly.

**Comment by BlackSmith** [comment]
OK, I removed Arduino configuration.

**Comment by 1technophile** [comment]
Thanks

**Comment by Dattel** [comment]
Does that feature also work with the new CC1101 feature?

**Comment by 1technophile** [comment]
@orrpan any idea on @Dattel question?

**Comment by orrpan** [comment]
@Dattel @1technophile no, the weathermodule uses its own RF functions (I think) not rc-switch/newswitch as RF/RF2. But I can look at it, I don’t have a weather station to test with

**Comment by Dattel** [comment]
hmmm.... okay no worries... I was just a question.. 
I wired some RBX6 and build a omqttg with weatherplugin but it only gathers crap, since my sensors are talking a different protocol..... so currently no use for me :-(

**Pull request merged**